<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardController</class>
 <widget class="QWizard" name="WizardController">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>412</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>564</width>
    <height>412</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowIcon">
   <iconset resource="pixmap.qrc">
    <normaloff>:/pixmap/image/icon.png</normaloff>:/pixmap/image/icon.png</iconset>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWizardPage" name="wizardPageRootPath">
   <property name="subTitle">
    <string/>
   </property>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#43a2db</string>
    </property>
    <property name="text">
     <string>&amp;添加...</string>
    </property>
   </widget>
   <widget class="QLabel" name="timelabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>21</height>
     </size>
    </property>
    <property name="text">
     <string>更新时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="btn_close">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-3</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wizardLabel_2">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>-7</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#43a2db</string>
    </property>
    <property name="text">
     <string>天气配置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelCreate_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>21</height>
     </size>
    </property>
    <property name="text">
     <string>分钟</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCreate_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>21</height>
     </size>
    </property>
    <property name="text">
     <string>地点：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#43a2db</string>
    </property>
    <property name="text">
     <string>&amp;删除...</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>74</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPageFinish">
   <widget class="QLabel" name="btn_close_2">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-3</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCompleteSet">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>87</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置完成</string>
    </property>
   </widget>
   <widget class="QLabel" name="wizardLabel">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>-7</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#43a2db</string>
    </property>
    <property name="text">
     <string>天气配置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pixmap.qrc"/>
 </resources>
 <connections/>
</ui>
